src/bb-functions-publish.ts

```diff
@@ -21,16 +21,25 @@ program
   .name('bb functions publish')
   .option('-b, --bump', 'Bump the revision number.')
   .option('-s, --skip', 'Skip building the custom functions bundle.')
+  .option(
+    '-h, --host <host>',
+    'Set hostname to publish to. Defaults to <identifier>.bettyblocks.com',
+  )
   .parse(process.argv);
 
 const bumpRevision = program.bump;
 const skipBuild = program.skip;
+const { host } = program;
 
 /* execute command */
 
 const workingDir = process.cwd();
 const identifier = acquireFunctionsProject(workingDir);
 
+const targetHost =
+  host ||
+  'https://{IDENTIFIER}.bettyblocks.com'.replace('{IDENTIFIER}', identifier);
```

Comment:
`https://${identifier}.bettyblocks.com` you can use this if you do it with backticks

Reply:
Yeah, mea culpa. Just copied it willy-nilly. 
I know about the backtick syntax. Not sure why I didn't use it here.

Clean it up!